diesel/src/pg/types/json.rs

```diff
@@ -76,14 +76,14 @@ fn some_json_from_sql() {
 fn bad_json_from_sql() {
     let uuid: Result<serde_json::Value, _> =
         FromSql::<sql_types::Json, Pg>::from_sql(Some(PgValue::for_test(b"boom")));
-    assert_eq!(uuid.unwrap_err().description(), "JSON error");
+    assert_eq!(uuid.unwrap_err().to_string(), "JSON error");
```

Comment:
This shows `serde_json`'s detail error so assertion failed. We should tweak the right value also.